source/contracts/reporting/FeeWindow.sol

```diff
@@ -80,13 +79,11 @@ contract FeeWindow is DelegationTarget, VariableSupplyToken, Extractable, Initia
     }
 
     function redeemForReportingParticipant() public returns (bool) {
-        redeemInternal(msg.sender, true);
-        return true;
+        return redeemInternal(msg.sender, true);
```

Comment:
I'd prefer these stay as is actually. Making it this way implies there's some information being passed.